packages/progress-bar/docs/ProgressBar.stories.tsx

```diff
@@ -1,96 +1,83 @@
 import React from "react"
+import { Story } from "@storybook/react"
 import { CATEGORIES } from "../../../storybook/constants"
 import { figmaEmbed } from "../../../storybook/helpers/figmaEmbed"
 import { ProgressBar } from "../index"
+import { StoryWrapper } from "../../../storybook/components/StoryWrapper"
 
 export default {
-  title: `${CATEGORIES.components}/ProgressBar`,
+  title: `${CATEGORIES.components}/Progress Bar`,
   component: ProgressBar,
   parameters: {
     docs: {
       description: {
         component: 'import { ProgressBar } from "@kaizen/progress-bar"',
       },
     },
-  },
-}
-
-export const DefaultStory = () => (
-  <ProgressBar
-    value={25}
-    max={100}
-    mood="positive"
-    isAnimating={true}
-    label="25%"
-  />
-)
-DefaultStory.story = {
-  name: "Positive (Kaizen Site Demo)",
-  parameters: {
     ...figmaEmbed(
       "https://www.figma.com/file/eZKEE5kXbEMY3lx84oz8iN/%E2%9D%A4%EF%B8%8F-UI-Kit%3A-Heart?node-id=1929%3A20890"
     ),
   },
 }
 
-export const PositiveSubtext = () => (
+export const DefaultStory = () => (
```

Comment:
```suggestion
export const DefaultStory: ComponentStory<typeof ProgressBar> = args => (
  <ProgressBar {...args} />
)
DefaultStory.storyName = "Default (Kaizen Demo)"
DefaultStory.args = {
  value: 25,
  max: 100,
  mood: "positive",
  isAnimating: true,
  label: "25%",
}
```

Couldn't highlight the whole section so the suggestion looks weird.
Realised there weren't any controls for ProgressBar, so this will add it in while also binding the values into them :)